gazebo/PKGBUILD

```
@@ -25,24 +26,21 @@ optdepends=('bullet: Bullet support'
             'libusb: USB peripherals support'
             'simbody: Simbody support'
             'urdfdom: Load URDF files')
-makedepends=('cmake' 'git' 'doxygen' 'ruby-ronn')
+makedepends=('cmake' 'doxygen' 'ruby-ronn')
 install="${pkgname}.install"
-source=("git+https://github.com/osrf/gazebo.git#tag=gazebo11_$pkgver")
-sha256sums=('SKIP')
+source=("${pkgname}.tar.gz::https://github.com/osrf/gazebo/archive/gazebo11_$pkgver.tar.gz"
```

```suggestion
source=("${pkgname}-${pkgver}.tar.gz::https://github.com/osrf/gazebo/archive/${pkgname}11_$pkgver.tar.gz"
```

We want to keep the version number in the filename, otherwise when the user upgrades they'll get an error due to mismatching checksums.

---

nice catch